src/screens/profileView/profileView.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { FormControl, FormControlState } from '@mui/base/FormControl';
+import Avatar from '@mui/material/Avatar';
+import List from '@mui/material/List';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import ListItemText from '@mui/material/ListItemText';
+
+import {
+  Wrapper,
+  ChangePasswordButton,
+  EditButton,
+  EditButtonWrapper,
+  AvatarWrapper,
+  Item,
+  StyledInput,
+  ChangePasswordForm,
+  SaveChangesButton,
+} from './profileViewStyles';
+
+export const ProfileView = () => {
+  const [showPasswordForm, setShowPasswordForm] = React.useState(false);
+
+  function generate(element: React.ReactElement) {
+    return [0, 1, 2, 3, 4].map((value) =>
+      React.cloneElement(element, {
+        key: value,
+      }),
+    );
+  }
```

Comment:
jeśli chcesz w ten sposób robić generate to będzie on tylko kopiować dane które statycznie wpiszesz poniżej. Użyj go aby zamiast tego tworzyło nowe elementy bazując na podanym array'u np. 
`const Grupy: grupy[] = [
  { zdjecie: 'zdjecie.png', nazwa_gurpy: 'informatyka stosowana', ilosc_czlonkow: 25 },
];`.